### PR TITLE
[feature][chunjun-connector/chunjun-connector-nebula] add a new conne…

### PR DESCRIPTION
**Pull request by gaoasi**
…ctor nebula

## Purpose of this pull request
add a new connector nebula

## Which issue you fix
Fixes # (issue).

## Checklist:

- [x] I have executed the **'mvn spotless:apply'** command to format my code.
- [x] I have a meaningful commit message (including the issue id, **the template of commit message is '[label-type-#issue-id][fixed-module] a meaningful commit message.'**)
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have checked my code and corrected any misspellings.
- [x] My commit is only one. (If there are multiple commits, you can use 'git squash' to compress multiple commits into one.)

### CODE REVIEW

File: chunjun-connectors/chunjun-connector-nebula/src/main/java/com/dtstack/chunjun/connector/nebula/client/NebulaSession.java [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+import com.vesoft.nebula.client.graph.net.Session;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.Serializable;
+import java.util.Locale;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static com.dtstack.chunjun.connector.nebula.utils.NebulaConstant.*;
```
Comment (FlechazoW) [inline chunjun-connectors/chunjun-connector-nebula/src/main/java/com/dtstack/chunjun/connector/nebula/client/NebulaSession.java:37]: Do not import *.

**Comment by FlechazoW** [comment]
Thank you so much, and we are so happy to review this PR. Before merging it, I have some suggestions about it.  Could you 
please change this pr base on the branch of '1.16_alpha'? We will merge the branch of '1.16_alpha' into the master branch as soon as we finish the test.

**Comment by gaoasi** [comment]
> Thank you so much, and we are so happy to review this PR. Before merging it, I have some suggestions about it. Could you please change this pr base on the branch of '1.16_alpha'? We will merge the branch of '1.16_alpha' into the master branch as soon as we finish the test.

sure,i will change the pr branch soon

**Comment by gaoasi** [comment]
> Thank you so much, and we are so happy to review this PR. Before merging it, I have some suggestions about it. Could you please change this pr base on the branch of '1.16_alpha'? We will merge the branch of '1.16_alpha' into the master branch as soon as we finish the test.

hi,i try to run a stream sql script demo with 1.16_alpha branch locally,but i got a exception :
image
do you know why?

**Pull request closed without merging**